manifests/init.pp

```diff
@@ -36,59 +36,13 @@
   Boolean $create_rndrelease = true,
   Boolean $create_symlink = true,
   Boolean $install_package = true,
-  Optional[Array[String[1]]] $packages = undef,
+  Array $packages = [],
```

Comment:
Should it still be Array[String[1]]?

Reply:
Ah. That is true.

Wouldn't it make sense though to ensure the correct input is used? Of course that would, again, force additional check on like 131. I guess I don't understand why the reason behind chaing this.
```
if $install_package == true and $packages {
```